app/presenters/tree_node_builder_compliance_history.rb

```diff
@@ -1,11 +1,15 @@
 class TreeNodeBuilderComplianceHistory < TreeNodeBuilder
+  def expand_node?
+    true if options[:open_all] && options[:expand] != false
```

Comment:
wondering if we can remove the `true if` part to make this a method always returning a boolean